packages/wrangler/package.json

```diff
@@ -171,11 +141,10 @@
 		"ink-table": "^3.0.0",
 		"is-ci": "^3.0.1",
 		"javascript-time-ago": "^2.5.4",
-		"jest": "^29.7.0",
-		"jest-websocket-mock": "^2.5.0",
 		"md5-file": "5.0.0",
 		"mime": "^3.0.0",
 		"minimatch": "^5.1.0",
+		"mock-socket": "^9.3.1",
```

Comment:
Noticing this added dep and `vitest-websocket-mock` – what's the reasoning for both?

Reply:
We'd been using it previously (just transitively through the dependencies of `jest-websocket-mock`). `mock-socket` is the client, and `vitest-websocket-mock` is the server